src/components/HomeLayout/Helpline.js

```diff
@@ -1,21 +1,77 @@
 import React, { useState } from "react";
 import { Modal, Row, Col, Button } from "antd";
 
-import { WomanOutlined } from "@ant-design/icons";
-import { Spacer } from "@components/Utils";
-import { helplineLinks } from "@utils/constants";
+// import { WomanOutlined } from "@ant-design/icons";
+// import { Spacer } from "@components/Utils";
+// import { helplineLinks } from "@utils/constants";
 
-const HelplineModal = ({ showModal, setShowModal }) => (
+const TravelAssistanceModal = ({ showModal, setShowModal }) => (
   <Modal
-    title={<div>Women&apos;s Helpline</div>}
+    title={<div>Travel Assistance</div>}
     visible={showModal}
     onOk={() => setShowModal(false)}
     onCancel={() => setShowModal(false)}
     footer={null}
   >
     <div style={{ textAlign: "left" }}>
       <div className="modal-txt">
-        <p> Helpline numbers are available in the links below, </p>
+        <p><a href="https://mapit.gov.in/covid-19" target="_new">
```

Comment:
Rather than rewriting the html code it would good to create an array of objects and iterate through it to generate this list dynamically. 
const travelLinks = [{ link: '...', label: '...'}, {link: '...', label: '...'}] , define this in constants and import it here.

in the code,
travelLinks.map( item => (
 <react_jsx>
));

Reply:
I have made changes as per the sugesstions